5.machine_learning/scripts/6.find_sc_crops_coeff.py

```diff
@@ -3,29 +3,29 @@
 
 # # Find one top representative single-cell crop visualize per top healthy and failing coefficients
 # 
-# The features with the highest coefficients per channel per cell type per identified *manually*. We do not use correlation features due to having to use two features and those are harder to visualize.
+# The features with the highest coefficients per channel per cell type are identified *manually*. We do not use correlation features due to having to use two features and those are harder to visualize.
 # 
 # We first filter the plate 4 data frame to only include isolated cells (0 cell neighbors adjacent) and then filter out any single-cell that is too close to an edge (based on crop_size).
 # 
 # ### **Top healthy features per channel**
 # 
-# | Feature | Coefficient |
-# |-----------------|-----------------|
-# | Nuclei_Intensity_MeanIntensityEdge_**Hoechst** | 1.6491632293619 |
-# | Cytoplasm_Texture_AngularSecondMoment_**ER**_3_01_256 | 0.251023308427374 |
-# | Nuclei_Intensity_IntegratedIntensity_**PM** | 0.647266129401344 |
-# | Nuclei_RadialDistribution_MeanFrac_**Mitochondria**_3of4 | 0.338620308838192 |
-# | Nuclei_Intensity_MinIntensity_**Actin** | 0.34104254666359 |
+# | Organelle | Feature | Coefficient |
+# |-----------------|-----------------|-----------------|
+# | Nucleus | Nuclei_Intensity_MeanIntensityEdge_Hoechst | 1.41883228668943 |
+# | ER | Nuclei_Texture_AngularSecondMoment_ER_3_01_256 | 0.247086864563417 |
+# | Golgi/Plasma membrane | Nuclei_Intensity_IntegratedIntensity_PM | 0.551412935417769 |
+# | Mitochondria | Cells_Intensity_MinIntensityEdge_Mitochondria | 0.247078061220504 |
+# | F-actin | Nuclei_Intensity_MinIntensity_Actin | 0.623372719535748 |
 # 
 # ### **Top failing features per channel**
 # 
-# | Feature | Coefficient |
-# |-----------------|-----------------|
-# | Nuclei_Intensity_StdIntensityEdge_**Hoechst** | 0.731721019099269 |
-# | Cells_Intensity_IntegratedIntensityEdge_**ER** | 0.260370773712069 |
-# | Cells_Texture_InfoMeas1_**PM**_3_00_256 | 0.277759767032262 |
-# | Nuclei_Intensity_LowerQuartileIntensity_**Mitochondria** | 0.280095597808861 |
-# | Cells_Intensity_IntegratedIntensityEdge_**Actin** | 1.1894472062255 |
+# | Organelle | Feature | Coefficient |
+# |-----------------|-----------------|-----------------|
+# | Nucleus | Cells_Intensity_StdIntensityEdge_Hoechst | 0.736873911951805 |
+# | ER | Cytoplasm_Texture_InverseDifferenceMoment_ER_3_01_256 | 0.357607115585286 |
+# | Golgi/Plasma membrane | Cytoplasm_Texture_InfoMeas1_PM_3_00_256 | 0.518741372506253 |
+# | Mitochondria | Nuclei_Intensity_IntegratedIntensity_Mitochondria | 0.26804586648328 |
+# | F-actin | Cells_Intensity_IntegratedIntensityEdge_Actin | 1.35720412209369 |
```

Comment:
These tables look really useful and like they could become table plots. Would it make sense to create them programmatically and export to PNG's like the others (elevating their importance and reuse)?

Reply:
Maybe combining the table with another plot could help too - something akin to the example found on  https://plotly.com/python/table-subplots/.

Reply:
I think this is a bit beyond the scope of this PR. I agree that this would be cool, so I will add this to my notes!